### DISCUSSION TITLE
Logging in to Bentley keeps displaying the "Accept" popup

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/community#53] [Q&A] [answered]
Hello I develop the FutureOn connector to iTwin which is used by Shell for their Bentley Plantsight projects.
The connector uses a OAUTH2 sign in process to sign in to Bentley.

Recently in the last few days, the sign in process keeps displaying the "Accept" popup dialogue.
See screenshot below.

<img width="569" alt="Screenshot 2024-10-09 at 11 15 52" src="https://github.com/user-attachments/assets/b6679933-626d-45e6-80b0-5f6b984d6919">

This is a change that didn't used to happen. So I am wondering why it keeps popping up every time?

I am wondering if there has been a change to the OAUTH2 scopes that my application uses?
The list of scopes which it currently uses are as follows:

```
scope: 'email openid profile organization itwinjs synchronization:modify synchronization:read transformations:read transformations:modify imodels:read imodels:modify storage:modify storage:read itwins:read itwins:modify changedelements:read changedelements:modify issues:read issues:modify'
```

**Comment by PeterJBassett** [comment]
As a follow up, I have tried deleting the deprecated scopes from the OAUTH2 sign in code.
So now the list is just:
  scope: 'email openid profile organization itwinjs itwin-platform',

This made no difference, the popup "Accept" screen keeps occuring each time.

**Reply by aruniverse** [reply]
tangential, but you shouldn't need the following scopes anymore, `email openid profile organization itwinjs` , just need `itwin-platform`

**Reply by PeterJBassett** [reply]
Ok, thanks for letting me know.

**Comment by hl662** [comment]
Hi Peter, I wanted to confirm with you that the client id you created here is for a third party right?

**Reply by PeterJBassett** [reply]
The client id is spa-m19YIiH1EpMzsd9RYBGa9jIJL it is for a webapp which I have developed which runs in an iframe in our FieldTwin web application. It is used by Shell to sync with their Bentley PlantSight projects.

**Reply by PeterJBassett** [reply]
Is this issue something to do with this?
https://bentleysystems.service-now.com/community?id=kb_article_view&sysparm_article=KB0038629

**Comment by hl662** [comment, marked as answer]
Hey Peter, we'll fix the issue shortly, and you should see the consent pop up disappear within today or tomorrow. If you update your client (either update the scopes, create a new client id, change the owners), you may find that the consent page prompts reappears. Let us know, and we'll disable it again. This is just in the short term - we have solutions planned in the long term so this doesn't happen and you won't have to notify us in future cases

**Reply by PeterJBassett** [reply]
Hello @hl662 thank you, the accept popup has now stopped happening. Thank you for the response.

**Reply by PeterJBassett** [reply]
Hello @hl662 this is just a note to let you know that the "Accept" popup is occuring again every time I log in. 
I have been working on getting my applicxation log in working, and so perhaps have changed something that has caused this problem to re-occur.

**Reply by hl662** [reply]
Thanks peter - I'll update you again when we've resolved the issue 👍 , expect it resolved early next week

**Reply by hl662** [reply]
Hey Peter, expect the consent page to disappear within the next two days. Thanks for your patience 😃
